SimPEG/EM/Static/SIP/Regularization.py

```diff
@@ -9,7 +9,7 @@
 from SimPEG.Regularization import BaseRegularization, Simple
 
 
-class MultiRegularization(Simple):
+class MultiRegularization(BaseRegularization):
```

Comment:
@sgkang: we can pull this out in a separate pr and use the combo objective functions. There is a small example of how you can do that in the tests: https://github.com/simpeg/simpeg/blob/ref/objectivefunctions/tests/base/test_regularization.py#L227